src/App.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div id="app">
+    <header>
+      <h1>Birdie Stats</h1>
+    </header>
+    <button id="btn-input-round" @click="isInputting = true" 
+    v-bind:disabled="isInputting">
+      Input New Round
+    </button><br><br>
+    <RoundForm v-if="isInputting"
```

Comment:
[v-if instead of v-show](https://vuejs.org/v2/guide/conditional.html#v-if-vs-v-show) since this shouldn't be toggled that often (users will likely only input 1 round when they visit the page)